integtest/spec/helper/dest.rb

```diff
@@ -39,9 +45,13 @@ def convert_all(conf)
       --target_repo #{bare_repo}
       --conf #{conf}
     ]
-    out = run_convert cmd
+    @convert_outputs << run_convert(cmd)
+  end
+
+  ##
+  # Checks out the results of the last call to convert_all
+  def checkout_conversion
```

Comment:
The new tests run `convert_all` multiple times and only need to checkout after the last time.